spec/services/idv/vendor_result_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe Idv::VendorResult do
+  let(:success) { true }
+  let(:errors) { { foo: ['is not valid'] } }
+  let(:reasons) { %w[foo bar baz] }
+  let(:session_id) { SecureRandom.uuid }
+  let(:normalized_applicant) do
+    Proofer::Applicant.new(
+      last_name: 'Ever',
+      first_name: 'Greatest'
+    )
+  end
+
+  subject(:vendor_result) do
+    Idv::VendorResult.new(
+      success: success,
+      errors: errors,
+      reasons: reasons,
+      session_id: session_id,
+      normalized_applicant: normalized_applicant
+    )
+  end
+
+  describe '#success?' do
+    it 'is the success value' do
+      expect(vendor_result.success?).to eq(success)
+    end
+  end
+
+  describe '#to_json' do
```

Comment:
Hmm, maybe it the purpose is to check that it correctly serializes things that don't translate to JSON primitives, it could be something like:

```ruby
it 'serializes normalized_applicant correctly'
```